reader_core/src/utils/sub_menu_capture.rs

```diff
@@ -0,0 +1,50 @@
+use super::CircularCounter;
+use crate::pnp;
+
+#[derive(Default)]
+pub struct SubMenuCapture<const MIN: usize, const MAX: usize> {
```

Comment:
It looks like this is a clone of the regular menu, but has the special `X` + up/down handling.

Is there a way we could use the normal menu and pull the special button combos into `run_frame`? That would prevent duplication.